src/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Grid</title>
+    <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+  </head>
+
+  <body>
+    <div class="containter">
+      <div class="containter__logo">
+        <span class="containter__logo-title">logo</span>
+      </div>
+      <header class="containter__header header">
+        <nav>
+          <ul class="header__list">
+            <li class="header__item">
+              <a class="header__link" href="#">home</a>
+            </li>
+            <li class="header__item">
+              <a class="header__link" href="#">about</a>
+            </li>
+            <li class="header__item">
+              <a class="header__link" href="#">gallery</a>
+            </li>
+          </ul>
+        </nav>
+        <a class="header__link" href="#">login</a>
+      </header>
+      <aside class="containter__sidebar sidebar">
+        <p class="sidebar__menu">menu</p>
```

Comment:
Better use <span> as it is not a paragraph

Reply:
ok